test/unit_test/static/quantity_point_test.cpp

```diff
@@ -22,8 +22,12 @@
 
 #include <units/quantity_point.h>
 #include "test_tools.h"
+<<<<<<< HEAD
 #include "units/math.h"
 #include <units/bits/common_quantity.h>
+=======
+#include <units/bits/common_type.h>
+>>>>>>> origin/master
```

Comment:
There are conflict markers throughout.

Reply:
Uh oh - haven't looked at the review diff at all yet - I did just run the tests locally. From the success, I assumed that I fixed all conflicts. I was wrong obviously.